trustmanager/x509utils.go

```diff
@@ -440,6 +451,11 @@ func EncryptPrivateKey(key data.PrivateKey, passphrase string) ([]byte, error) {
 		return nil, err
 	}
 
+	if encryptedPEMBlock.Headers == nil {
```

Comment:
I'm inclined to agree. This felt safer though as `EncryptPEMBlock` returns a specific error and if for some reason `Headers` isn't initialized attempting to assign will cause a panic. I can certainly error instead of initializing, that just felt presumptive give the explicit error return.